src/auth.js

```diff
@@ -170,22 +169,7 @@ export function buildAuthEntry(envelope, signature, publicKey) {
         address: new Address(publicKey).toScAddress(),
         nonce: auth.nonce(),
         signatureExpirationLedger: auth.signatureExpirationLedger(),
-        signatureArgs: [
-          nativeToScVal(
-            {
-              public_key: StrKey.decodeEd25519PublicKey(publicKey),
-              signature
-            },
-            {
-              // force the keys to be interpreted as symbols (expected for
-              // Soroban [contracttype]s)
-              type: {
-                public_key: ['symbol', null],
-                signature: ['symbol', null]
-              }
-            }
-          )
-        ]
+        signature: nativeToScVal(signature)
```

Comment:
TODO: Confirm that this is the right thing to do ([Slack thread](https://stellarfoundation.slack.com/archives/C030Z9EHVQE/p1691174074687949)).

Reply:
narrator: "it wasn't"